.storybook/main.ts

```diff
@@ -1,26 +1,17 @@
+import type { StorybookConfig } from "@storybook/react-webpack5";
 import path from "path";
 import remarkGfm from "remark-gfm";
 
-module.exports = {
-    stories: [
-        "../stories/**/!(*.stories).mdx",
-        "../stories/**/*.stories.@(ts|tsx)",
-    ],
+const config: StorybookConfig = {
+    stories: ["../stories/**/*.mdx", "../stories/**/*.stories.@(ts|tsx)"],
     addons: [
+        "@storybook/addon-webpack5-compiler-swc",
         "@storybook/addon-links",
         "@storybook/addon-essentials",
         "@storybook/addon-a11y",
         "@storybook/addon-themes",
-        {
-            name: "@storybook/addon-docs",
-            options: {
-                mdxPluginOptions: {
-                    mdxCompileOptions: {
-                        remarkPlugins: [remarkGfm],
```

Comment:
this was needed to handle markdown for a specific syntax, but I can't remember what exactly

Reply:
![image](https://github.com/LifeSG/react-design-system/assets/71387583/03d2dd7a-499e-4394-9ac7-706f31b96e58)
yes gets this error

Reply:
ok figured out, it's required for tables to work in mdx http://localhost:6006/?path=/docs/modules-errordisplay--docs

is there a new version that's compatible with SB8?

Reply:
seems like the v3 -> v4 fixes this